eng/pipelines/optprof-scheduled.yml

```diff
@@ -0,0 +1,149 @@
+# Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+# Name: DotNet-Project-System
+# URL: https://dev.azure.com/devdiv/DevDiv/_build?definitionId=9675
+
+# Produces the signed product build for insertion into the Visual Studio & Visual Studio Code Extension
+
+###################################################################################################################################################################
+# PIPELINE METADATA
+###################################################################################################################################################################
+
+# Activates the pipeline after a PR is merged.
+# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/trigger?view=azure-pipelines
+trigger:
+  branches:
+    include:
+    - main
+    # Our VS alignment branches start with dev (for example: dev17.2.x)
+    - dev*
+    # Any other branches that contain major feature development, or require access to the pipeline, prior to merge.
+    - feature/*
+  paths:
+    exclude:
+    - docs/*
+    - README.md
+
+# Disable PR builds for this pipeline.
+# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines#examples
+pr: none
+
+schedules:
```

Comment:
How do the `trigger` section and `schedules` section interact? Does the pipeline run on a schedule, but only if there are changes that match the branches and paths?

Reply:
See above comment, the trigger section can now be safely removed. We only want to run OptProf on a schedule